src/errorpage.cc

```diff
@@ -148,13 +148,12 @@ static void ValidateStaticError(const int page_id, const SBuf &inputLocation);
  \note  hard coded error messages are not appended with %S
  *      automagically to give you more control on the format
  */
-static const struct {
+struct DefaultMessages {
     int type;           /* and page_id */
     const char *text;
-}
-
-error_hard_text[] = {
+};
 
```

Comment:
Let's explain what this array is about.

```suggestion

/// error messages that cannot be configured/customized externally
```

Reply:
Done.